### DISCUSSION TITLE
Dragging sidebar panel should move window like vertical tab panel

### DISCUSSION THREAD

**Discussion by cgarst** [Floorp-Projects/Floorp#1212] [Feature Request]
In the vertical tab configuration, the browser appears fairly symmetrical with the tabs on the left and sidebar on the right. The user can drag the empty space in the vertical tab bar to move the window. However the user cannot drag from the empty space in the sidebar. It would be more ideal to allow this on both sides for intuitive symmetrical behavior.

**Comment by ohnomonkey** [comment]
This would be a tremendous quality-of-life improvement, especially when hiding the title and tab bars for a pure minimalist config. 🧘
